data/tutorials/getting-started/gs_01_a_tour_of_ocaml.md

```diff
@@ -51,21 +51,21 @@ utop #
 
 Press `Ctrl-D` (end of file) or enter `#quit;;` to exit `utop`.
 
-The OCaml code examples in this tutorial show the input needed at the hash prompt `#`, as displayed by `utop`, ending with double semicolumn `;;`. The `;;` let OCaml know the expression is complete. Code sample lines beginning with dollar sign `$`, like above, should be entered in an terminal.
+The hash prompt `#` displayed by UTop means it is waiting for input. You can start writing your code after the prompt. To evaluate it, add a double semicolon `;;` to signal the end of the expression and press `Enter`.
 
-Upon hitting `Enter`, OCaml runs the code. The following line shows the output. For simplicity, anything but the user input and OCaml's output in `utop` is omitted in this tutorial.
+For instance, consider the following code snippet:
 ```ocaml
 # 2 + 2;;
 - : int = 4
 ```
 
-Neither the prompt `#` nor the double semicolumns `;;` are part of the code. When the toplevel displays the prompt `#`, it means it is waiting for user input. The double semicolumn means “send.” It marks the end of the input and signals OCaml to run the code. Without double semicolumn, pressing return just creates a new line.
+In the code snippet above, `2 + 2;;` is the user's input, and `- : int = 4` is the output of OCaml.
 
-Again, the line which follows shows OCaml's result. The above example says, “What you've entered is an integer (`int`) that evaluates into 4.” Since it is anonymous expression, the character `-` instead of a name.
+If you need to amend the code before hitting `Enter`, you can use your keyboard's right and left arrows to move inside the text. The up and down arrows allows navigation through previously evaluated expressions. Typing `Enter` without a double semicolon `;;` will create a new line: you can write multiple-line expressions this way.
 
-If you need to amend the code before hitting enter, you can use your keyboard's right and left arrows to move inside the text. The up and down arrows allows navitation through previously typed commands, if you have multiple lines of code.
+Commands beginning with a dash character `#` such as `#quit` or `#help` are not evaluated by OCaml: they are interpreted as commands by UTop.
 
-Remember: commands beginning with a dash character `#` such as `#quit` or `#help` are not evaluated by OCaml, they are interpreter by Utop.
+You're now ready to hack with UTop and follow through this Tour of OCaml. If you hit any issue with the toplevel, don't hesitate ???
```

Comment:
```suggestion
You're now ready to hack with UTop and follow along through this Tour of OCaml. If you hit any issue with the toplevel, don't hesitate ???
```
Minor suggestion, feel free to ignore. I added "along" in here because I read it as "follow through" as to do something you've promised. 

..."don't hesitate to contact us" ?